src/services/job-role-service.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Job Role Service for handling API calls to job-roles endpoints
+ */
+
+import { promises as fs } from "node:fs";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import axios from "axios";
+import type { JobRoleResponse } from "../models/job-role-response.js";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+export interface IJobRoleService {
+	getJobRoles(): Promise<JobRoleResponse[]>;
+}
+
+export class JobRoleService implements IJobRoleService {
```

Comment:
is, do include the information about this service using json file (instead of calling the API) in the class name